packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -413,6 +420,11 @@ export class GraphWidget extends ConcreteWidget {
       ...(this.props.leftAnnotations || []).map(mapAnnotation('left')),
       ...(this.props.rightAnnotations || []).map(mapAnnotation('right')),
     ];
+    const verticalAnnotations = this.props.verticalAnnotations || [];
```

Comment:
This line doesn't seem needed. When you check against `verticalAnnotations.length` you could instead just check against `this.props.verticalAnnotations?.length > 0`.